api/src/main/java/com/microsoft/gctoolkit/io/RotatingGCLogFile.java

```diff
@@ -47,22 +47,22 @@ private static boolean isUnifiedLogging(Path path) {
             } else {
                 segments = findGCLogSegments(path);
             }
-            return isUnifiedLogging(path, segments);
+            return isUnified(path, segments);
         } catch (IOException e) {
             LOGGER.log(Level.WARNING, "cannot determine whether " + path + " is a unified log format");
             return false;
         }
     }
 
-    private static boolean isUnifiedLogging(Path path, List<GarbageCollectionLogFileSegment> segments) {
+    private static boolean isUnified(Path path, List<GarbageCollectionLogFileSegment> segments) {
         // TODO: if isUnifiedLogging is false, assert that the file is pre-unified.
         //       if file is neither unified nor pre-unified, then we're dealing with
         //       something we can't handle.
         return segments.stream()
                 .map(GarbageCollectionLogFileSegment::stream)
                 .anyMatch(s -> {
```

Comment:
I'm still torn between this single character convention inside lambdas.  I know Oracle started this trend when they first introduced these things but I'm always haveing to go back and go 'What's an s again?'

Reply:
I don't think it's necessary for the GCLogFile isUnified(String<Stream>) method to throw IOException. If that's true, then this try-catch ugliness goes away.

Reply:
this is all currently being collapsed as it's logic that is repeated in a few places